stdlib/universe/window.go

```diff
@@ -124,6 +124,92 @@ func createWindowOpSpec(args flux.Arguments, a *flux.Administration) (flux.Opera
 	return spec, nil
 }
 
+func CreateWindowOpSpec(args flux.Arguments, a *flux.Administration) (flux.OperationSpec, error) {
```

Comment:
Why was this function added?

Reply:
That function was added because we need to export it to the experimental package and go requires an all uppercase function in order to export.

Reply:
Can you export the existing function instead of duplicating the function?